brainstormforce/astra
# AST-3018 - Accessibility Improvements for Woocommerce

### Description

- Accessibility Issues with Tab Navigation fixed.
- JIRA :- https://brainstormforce.atlassian.net/browse/AST-3018

### Screenshots

- Please refer here : https://brainstormforce.atlassian.net/browse/AST-3018

### Types of changes




Improvements

### How has this been tested?

- Set Global Highlight Type and Input Highlight Type from Customizer.
- Global Highlight Type will be accessible to complete site including input fields.
- Input Highlight Type will override Global Highlight Type for input fields.
- Check for each sub task, if elements are accessible through tab navigation as expected.

### Checklist:
